client/src/components/ChatWindow.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { makeStyles } from "@material-ui/core/styles";
+
+import ChatInput from "./ChatInput";
+
+const ChatWindow = (props) => {
+  const useStyles = makeStyles((theme) => ({
```

Comment:
You can actually move this outside of the function, just above line 7. I suggest doing this because it creates the object only once - leaving it in the function will create it over and over every render :) A bit better for performance

Reply:
ohhh I see.  I'll move all of the useStyles to outside of the component functions.